lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -374,15 +375,15 @@ pub struct SigStack {
 
 impl SigStack {
     pub fn new(sp: *mut libc::c_void, flags: SigStackFlags, size: libc::size_t) -> SigStack {
-        let mut stack = unsafe { std::mem::uninitialized::<libc::stack_t>() };
+        let mut stack: libc::stack_t = unsafe { mem::uninitialized() };
```

Comment:
We explicitly initialize all the fields of `libc::stack_t` here, so we don't actually need to use uninitialized memory at all.